lib/puppet/provider/base_dsc/powershell.rb

```diff
@@ -2,6 +2,8 @@
   confine :operatingsystem => :windows
   defaultfor :operatingsystem => :windows
 
+  require 'lib/puppet/file_system'
```

Comment:
nit - I think we usually keep these at the top of the file (even when the provider is confined) - see https://github.com/puppetlabs/puppet/blob/master/lib/puppet/provider/package/windows.rb for example